packages/fiori-app-sub-generator/test/unit/fiori-app-generator/writing.test.ts

```diff
@@ -118,7 +118,9 @@ describe('`writing` tests', () => {
                 template: 'Basic V4',
                 generatorName: '@sap/some-generator',
                 entityRelatedConfig: [],
-                launchText: t('readme.texts.runInstruction')
+                launchText: t('readme.texts.runInstruction'),
+                externalParameters: undefined,
```

Comment:
Unexpected new properties?

Reply:
I am not sure, but I see other tests have `externalParameters` too

Reply:
Of can you revert then, its not related to text changes